### ISSUE TITLE
Bug: Missing alt attributes on icon images in index.html

### ISSUE THREAD

**Issue by bushidocodes** [bushidocodes/froot-wars#44]
## Description

Two `<img>` elements in `index.html` are missing `alt` attributes, which is an accessibility violation:

```html
<!-- scorescreen: restart button has no alt -->
<img src="./images/icons/prev.png" id="restartlevel">

<!-- endingscreen: return button has no alt -->
<p id="showLevelScreen"><img src="./images/icons/return.png">Return to Level Screen</p>
```

Both images are interactive (they have click handlers attached via JS), so screen reader users need descriptive text to know their purpose.

## Fix

```html
<img src="./images/icons/prev.png" id="restartlevel" alt="Restart level">

<img src="./images/icons/return.png" alt="Return">
```

**Comment by bushidocodes** [comment]
Fixed in PR #56.
